backend/src/core/services/campaign.service.ts

```diff
@@ -144,25 +150,95 @@ const listCampaigns = ({
   userId,
   offset,
   limit,
+  type,
```

Comment:
same point re: the typing here https://github.com/opengovsg/postmangovsg/pull/1727/files#r944067275

probably should be in the shared folder imo